docs/compute-element/install-htcondor-ce.md

```diff
@@ -27,10 +27,6 @@ Before starting the installation process, consider the following points
 -   **Network ports:** The pilot factories must be able to contact your HTCondor-CE service on port 9619 (TCP)
 -   **Submit host:** HTCondor-CE should be installed on a host that already has the ability to submit jobs into your
     local cluster
--   **File Systems:** We recommend separate partitions for the [job queue log](#configuring-the-job-queue-location) and
```

Comment:
Can we keep this bullet point but delete the first sentence?

Reply:
Yeah, on a shared FS. So I was thinking something like this

```
**File Systems**:  Non-HTCondor batch systems require a [shared file system](#batch-systems-other-than-htcondor) between the HTCondor-CE host and the batch system worker nodes.
```

Reply:
Should we say anything about using local disk for SPOOL for an HTCondor batch system?

Reply:
I think it's implied but I guess it doesn't hurt to be explicit

Reply:
Hm. It would be too much info in the "Before Starting" section and I can't find another good place to put it. Let's postpone that for a later PR; maybe make a section for configuring _with_ an HTCondor batch system in the "Configuring HTCondor-CE" section.